sched/task/task_spawnparms.c

```diff
@@ -65,7 +65,9 @@ static inline int nxspawn_close(FAR struct tcb_s *tcb,
 
   sinfo("Closing fd=%d\n", action->fd);
 
-  return nx_close_from_tcb(tcb, action->fd);
+  nx_close_from_tcb(tcb, action->fd);
```

Comment:
why ignore the return value

Reply:
`nx_close_from_tcb()` may fail here because `SPAWN_FILE_ACTION_CLOSE` does not dup corresponding file.

Reply:
move this change to separate commit